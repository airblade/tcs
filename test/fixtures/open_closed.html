<details class="open:bg-white open:ring-black/5 dark:open:ring-white/10 p-6" open>
</details>
