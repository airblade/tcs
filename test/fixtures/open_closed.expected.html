<details class="p-6 open:bg-white open:ring-black/5 dark:open:ring-white/10" open>
</details>
